Content.Shared/Zombies/PassiveHealComponent.cs

```diff
@@ -0,0 +1,32 @@
+using Content.Shared.Damage;
```

Comment:
If it is applicable, there is now a `PassiveDamageComponent` that can be used to implement passive regen.